packages/headless/src/controllers/commerce/facets/core/headless-core-commerce-facet.test.ts

```diff
@@ -26,6 +26,7 @@ import {
   CoreCommerceFacet,
   CoreCommerceFacetOptions,
 } from './headless-core-commerce-facet';
+import * as HasActiveValuesDeterminor from './utils/headless-core-commerce-facet-utils';
```

Comment:
I decided to take this function out of the core facet controller because I didn't want to have conditional logic based on the facet type in the core controller itself.

I think doing that also makes the tests somewhat cleaner and more complete for this part.